.travis.yml

```diff
@@ -0,0 +1,74 @@
+language: python
+python:
+ - "3.6"
+ - "3.7"
+dist: bionic
+services:
+- docker
+
+env:
+ global:
+  - ICON_RC_REPO="https://github.com/icon-project/rewardcalculator.git"
+  - GOROOT="/snap/go/current"
+  - GOTOOLDIR="/snap/go/current/pkg/tool/linux_amd64"
+  - export PATH=/snap/bin:$PATH
+ matrix:
+  - TEST_TARGET=unit-test
+  - TEST_TARGET=integration-test
+
+matrix:
+ allow_failures:
+  - python: "3.6"
+
+addons:
+ apt:
+  update: true
+  packages:
+   - make
+   - build-essential
+   - libssl-dev
+   - zlib1g-dev
+   - libbz2-dev
+   - libreadline-dev
+   - libsqlite3-dev
+   - wget
+   - curl
+   - llvm
+   - libncurses5-dev
+   - libncursesw5-dev
+   - xz-utils
+   - tk-dev
+   - libffi-dev
+   - liblzma-dev
+   - automake
+   - libtool
+   - lsof
+   - figlet
+   - libsecp256k1-dev
+
+ snaps:
+  - name: go
+    confinement: classic
+ 
+before_install:
+ - git clone $ICON_RC_REPO;
+ - cd rewardcalculator;
+ - git checkout develop -f;
+ - make && sudo make install
+ - cd -;
+ - icon_rc -version;
```

Comment:
remove semicolon?!

Reply:
Thank you!
I tested by multi-line command and forgot to remove them.